app/src/main/java/com/mensinator/app/NotificationReceiver.kt

```diff
@@ -9,11 +9,12 @@ import androidx.core.app.NotificationManagerCompat
 
 class NotificationReceiver : BroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
+        val messageText = intent.getStringExtra("messageText")
```

Comment:
extract `messageText` to a val somewhere (so that it is used across both "sides", ie also at NotificationScheduler)

Reply:
See commit https://github.com/EmmaTellblom/Mensinator/pull/210/commits/6199411876821c9575ee46e53010f6ca41d4776f :)